internal/resources/ekscluster/flatten.go

```diff
@@ -0,0 +1,301 @@
+/*
+Copyright 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package ekscluster
+
+import (
+	models "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/ekscluster"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
+)
+
+func flattenClusterSpec(item *models.VmwareTanzuManageV1alpha1EksclusterSpec) []interface{} {
```

Comment:
Can we have two resource files made, one for cluster and another for node pools and add the construct and flatten functions respectively in each of them. That will make it in sync with how other resources have been implemented.

Reply:
done